src/lib/containers/entity_finder/details/configurations/FavoritesDetails.tsx

```diff
@@ -10,7 +10,7 @@ export const FavoritesDetails: React.FunctionComponent<
   FavoritesDetailsProps
 > = ({ ...sharedProps }) => {
   const { data, isLoading, hasNextPage, fetchNextPage, isFetchingNextPage } =
-    useGetFavoritesInfinite({ useErrorBoundary: true })
+    useGetFavoritesInfinite('NAME', 'ASC', { useErrorBoundary: true })
 
   const entities = data?.pages.flatMap(page => page.results) ?? []
 
```

Comment:
Below, we could add functionality to allow the user to change the direction of the sort as in other Entity Finder configurations, but would require some refactoring since it currently expects all of `name`, `createdOn`, `modifiedOn` to be sortable when in this case only `name` is.